src/components/formio/fieldset.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import {expect} from '@storybook/jest';
+import {Meta, StoryFn, StoryObj} from '@storybook/react';
+import {within} from '@storybook/testing-library';
+
+import {withFormik} from '@/sb-decorators';
+
+import Fieldset from './fieldset';
+
+export default {
+  title: 'Formio/Containers/Fieldset',
+  component: Fieldset,
+  decorators: [withFormik],
+  parameters: {
+    modal: {noModal: true},
+    // https://github.com/bbbtech/storybook-formik/issues/51#issuecomment-1136668271
+    docs: {
+      inlineStories: false,
+      iframeHeight: 150,
+    },
+  },
+  args: {
+    type: 'textfield',
+  },
+} as Meta<typeof Fieldset>;
+
+type Story = StoryObj<typeof Fieldset>;
+
+const Template: StoryFn<typeof Fieldset> = args => (
+  <Fieldset {...args}>
+    <div>{args.children || '<any children />'}</div>
+  </Fieldset>
+);
```

Comment:
I think you can drop this entirely and just set the `children` arg default on the `default` export to `<any children />` string